app/policies/project_policy.rb

```diff
@@ -28,7 +28,7 @@ def resolve
       elsif is_admin?
         current_team.projects
       else
-        current_user.projects.not_archived.where(id: current_team.projects.pluck(:id))
+        current_user.projects.not_archived.where(id: current_team&.projects&.pluck(:id))
```

Comment:
I think we can use explicit `return` statement in this case @talyssonoc 

```ruby
return [] if current_team

current_user
  .projects
  .not_archived
  .where(id: current_team.projects.pluck(:id))
```

What do you think?